_posts/2016-05-03-scaling-cto-talk.markdown

```diff
@@ -0,0 +1,17 @@
+---
+layout:       post
+title:        The Scaling CTO - Talk from the Wheelhouse CTO Summit at the NASDAQ
+author:       on_freund
+summary:
+image:        //res.cloudinary.com/wework/image/upload/c_lfill,f_auto,fl_progressive,g_north,h_1400,w_1400/v1461959537/engineering/scaling_cto_talk.jpg
+categories:   engineering process culture
+wistia_video: wistia_async_sbkrpcark4
+---
+
```

Comment:
You ready to shill???? Can you give more context? Like a tl;dr paragraph with what it's generally about (SEO JUICE!)